mobile-center-crashes/ios/RNCrashes/RNCrashes.m

```diff
@@ -105,7 +105,12 @@ - (NSDictionary *)constantsToExport
 {
     void (^fetchLastSessionCrashReport)() = ^void() {
         MSErrorReport *report = [MSCrashes lastSessionCrashReport];
-        resolve(convertReportToJS(report));
+        NSDictionary *jsonReport = convertReportToJS(report);
```

Comment:
@guperrot Good point, my new commit actually fixed a corner case bug. I looked at [the other reference](https://github.com/Microsoft/mobile-center-sdk-react-native/blob/master/mobile-center-crashes/ios/RNCrashes/RNCrashesDelegate.m#L48) to `convertReportToJS` method, and found `sendAppEventWithName` method is expecting `nil` instead of an empty `NSDictionary` in the case of null parameter.